src/pcrlock/pcrlock.c

```diff
@@ -1959,7 +1959,7 @@ static char *color_for_pcr(EventLog *el, uint32_t pcr) {
         if (el->registers[pcr].color)
                 return el->registers[pcr].color;
 
-        hsv_to_rgb(360.0 / (TPM2_PCRS_MAX - 1) * pcr, 100, 90, &r, &g, &b);
+        hsv_to_rgb(90.0 / (TPM2_PCRS_MAX - 1) * pcr + 180, 100, 90, &r, &g, &b);
```

Comment:
i am fine with removing green and red from the colors, but let's do this properly, i.e. add a small helper that avoids the ranges 0°-20°, 90°-150°, 340°-360° or so, see:

https://en.wikipedia.org/wiki/Hue#/media/File:HueScale.svg

i.e. something like:

```c
static double map_pcr_to_hue(uint32_t pcr) {
       /* Exclude red and green ranges, i.e. 340°-20° and 90°-150° */
       double h = 260.0 / (TPM2_PCRSS_MAX - 1) * pcr;
       h += 20; /* hide 0°…20° (red tones) */
       if (h > 90.0)
               h += 60.0; /* skip over 90°-150° (green tones) */
       assert(h <= 340);  /* this should avoid 340°-360° (red tones, again) */
       return h;
}
```

or something like this. completely untested, and the angles might need tweaking, but you get the idea

Reply:
not addressed?